tock/tock/management/commands/update_billable_expectations.py

```diff
@@ -40,7 +47,15 @@ def handle(self, *args, **options):
                         timecards = Timecard.objects.filter(user__username=username, reporting_period__start_date__gte=start_date)
                         self.stdout.write(f'Updating {len(timecards)} timecards for {username} to target hours {target}')
 
+                        target_billable_expectation = self._get_target_billable_expectation(target)
                         # Update all identified timecards
-                        [self._update_timecard(timecard, float(target)) for timecard in timecards]
+                        [self._update_timecard(timecard, target_billable_expectation) for timecard in timecards]
+
+                        if update_user_data:
```

Comment:
I think it might be worth pulling this out into an `_update_userdata` function to parallel the existing `update_timecard`.

Reply:
Thanks for the nudge!!! Added a commit with this tweak